lms/djangoapps/teams/static/teams/js/teams_tab_factory.js

```diff
@@ -1,13 +1,22 @@
 ;(function (define) {
     'use strict';
 
-    define(['jquery', 'teams/js/views/teams_tab'],
-        function ($, TeamsTabView) {
-            return function () {
-                var view = new TeamsTabView({
-                    el: $('.team-tab-content')
-                });
-                view.render();
-            };
-        });
+    define(['jquery',
+            'teams/js/views/teams_tab',
+            'js/components/header/views/header',
+            'js/components/header/models/header'],
+           function ($, TeamsTabView, HeaderView, HeaderModel) {
+               return function (headerInfo) {
+                   var headerModel = new HeaderModel(headerInfo),
+                       headerView = new HeaderView({
```

Comment:
I think that the HeaderView shouldn't be a top class entity, but instead something created by a view that wants a header. So in this case I'd expect the ``TeamsTabView`` to construct the header.